srcs/requirements/frontend/src/navigation.ts

```
@@ -17,23 +19,28 @@
  * @param replace A boolean indicating whether to replace the current history state or push a new one.
  */
 
-export function navigateTo(view: string, replace: boolean = false): void {
+export function navigateTo(view: string, update_history: boolean = true): void {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)